### ISSUE TITLE
SublimeREPL will not fully initiate R (sublime Text 3, Mac 10.9.5, R 3.2.2) 

### ISSUE THREAD

**Issue by jrzaurin** [wuub/SublimeREPL#454]
I basically use sublimeREPL for python, and it is just perfect.  I also code in R using either emacs or RStudio, and I though it would be nice to have all my code within the same text editor. Also, I found this new package R-Box which seems to be very nice for R-coding in sublime. 

I decided to give it a go but I find that, when I launch R, the loading process is incomplete. This is, I do: 

`Tools -> SublimeREPL -> R`

And I get the initialisation message: 

` R version 3.2.2 (2015-08-14) -- "Fire Safety"`
`Copyright (C) 2015 The R Foundation for Statistical Computing`
`Platform: x86_64-apple-darwin13.4.0 (64-bit)`

`R is free software and comes with ABSOLUTELY NO WARRANTY.`
`You are welcome to redistribute it under certain conditions.`
`Type 'license()' or 'licence()' for distribution details.`

`Natural language support but running in an English locale`

`R is a collaborative project with many contributors.`
`Type 'contributors()' for more information and`
`'citation()' on how to cite R or R packages in publications.`

`Type 'demo()' for some demos, 'help()' for on-line help, or`
`'help.start()' for an HTML browser interface to help.`
`Type 'q()' to quit R.`

`Revolution R Open 3.2.2`
`Default CRAN mirror snapshot taken on 2015-08-27`
`The enhanced R distribution from Revolution Analytics`
`Visit mran.revolutionanalytics.com/open for information`
`about additional features.`

`Multithreaded BLAS/LAPACK libraries detected. Using 8 cores for math algorithms. `

but NOT the arrow symbol to start coding `>` , and of course, I can't code in the R-console within sublime. 

I have tried to edit the `default_extend_env` to a series of options (some due to desperation):

`default_extend_env": {"PATH": "{PATH};/usr/bin/R"}`
`default_extend_env": {"PATH": "{PATH};/Library/Frameworks/R.framework/Versions/R"}`
`default_extend_env": {"PATH": "{PATH};/Applications/Revolution R Open.app"}`

but nothing seems to have any effect at all. R is fully functional in RStudio or emacs so I can keep coding there, but any help would be highly appreciated, since having all scripts in one editor would be ideal for me. 

Thanks in advance for your time.

**Comment by dschneiderch** [comment]
Yes r-box and sublimerepl works nicely (although you lose r-specific autocomplete and help functions). I had a similar issue and ended up having to download sublimerepl and r-box manually and extracting to the package to ~/Library/Application Support/Sublime Text 3/Packages and then reinstalling.

@jrzaurin you can also tell r-box to use terminal ( shift+cmd+p  then select r-box application). open terminal and start a r session and it should send the commands from st3 to the terminal r session.
hope that helps.
OSX 10.10.1, R 3.2.2

**Comment by jrzaurin** [comment]
@dschneiderch thank you!, although that looks like "a lot" of work (the manual installation, I mean). By the way, when you say "...and then reinstalling" what do you mean?

Anyway, I just wanted it because I could have all python and R in the same editor (split the window in 4 with python and R scripts and corresponding consoles), but for now emacs and ees is doing very well for R and I prefer not to touch the sublime set up :) unless is fairly easy.

The frustrating thing is that I made the whole installation in one of my team mates´s Mac (Yosemite) and works perfectly. 

Anyway, thanks for taking the time in replying!

**Comment by dschneiderch** [comment]
@jrzaurin It's not a lot of work. go to the github repo for each package and download the zip file. extract the zip file to the directory i mentioned and then go back to sublime and use shift+cmd+p "Install Packages" . For whatever reason, I couldn't see sublimerepl as an option to install until I manually added the package to the Packages directory.

I'm not sure exactly what functionality is from r-box and what is from sublimerepl so if python et al. are working then maybe you don't want to change anything. manually installing just r-box might help without affecting python. gl

**Comment by jrzaurin** [comment]
Hey sorry, I forgot to reply. 

I will give it a go, but note that the problem is not related to r-box but sublime, that will not initialise R. Therefore, I would have to install manually sublimeREPL. 
As you say, no big deal, but given that works with all my other languages...
Anyway, *many* thanks again.
